docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.less

```diff
@@ -0,0 +1,36 @@
+@import '../../../../../../assets/styles/variables.less';
+
+.btn-text {
+    color: #676a6c;
+    text-transform: none;
+    font-size: 16px;
+    font-weight: 400;
+}
+
+.icon-code-snippets-table {
+    width: 100%;
+    tr {
+        height: 30px;
+        td {
+            padding-top: 10px;
+            padding-bottom: 10px;
+            .ux-icon-clipboard {
+                color: @grey5;
+                cursor: pointer;
+                &:hover, &:focus {
+                    color: @grey1;
+                }
+            }
+        }
+    }
+}
+
+.usage-text {
+    padding-top: 10px;
+}
+
+.button-secondary.btn-link:hover,
+.btn-white.btn-link:hover {
+    width: 100%;
```

Comment:
Don't change the dimension on hover. Width should be 100% at all times for a consistent mouseover target. Also, you established a unique class for the icon buttons so please use that in the selector.

Reply:
Updated to whole element rather than hover